main/modes/games/ultimateTTT/ultimateTTTgame.c

```diff
@@ -593,42 +683,47 @@ static void tttPlaceMarker(ultimateTTT_t* ttt, const vec_t* subgame, const vec_t
     if (won || lost || drew)
     {
         // Record the outcome
-        if (won)
+        if (!ttt->game.singlePlayer)
         {
-            // Increment wins
-            ttt->wins++;
-            writeNvs32(tttWinKey, ttt->wins);
-            ttt->lastResult = TTR_WIN;
-
-            // Check for unlocked markers
-            for (int16_t mIdx = 0; mIdx < NUM_UNLOCKABLE_MARKERS; mIdx++)
+            // Don't give points for beating the computer!
```

Comment:
I was wondering about that. I'd probably allow unlocks against CPU on the principle that stuff should be unlockable without needing multiple Swadges.

Reply:
That's fair. Should it be for all difficulties, or just hard maybe?

Reply:
I'd tilt towards all difficulties. That way we don't punish young'ns who can play on easy. Mensa members can still play hard, and I don't think they'd mind (or notice) that easy also unlocks.

Reply:
Done!